docs/Agent/installation/ansible_based_installation.md

```diff
@@ -0,0 +1,37 @@
+# Ansible based installation
+
+In the following: we describe how to install Pbench Agent using an ANSIBLE playbook.
+
+:::{note}
+The same Pbench Agent version must be installed on all the test systems that participate in a benchmark run, there is no support for mixed installations.
+:::
+
+## Setup
+
+1. Make sure that you have the ANSIBLE package installed.
+
+2. Install the `pbench.agent` ANSIBLE collection from Ansible Galaxy.
+
+ ```console
+ ansible-galaxy collection install pbench.agent
+ ```
+
+3. Tell ansible where to find these roles.
+
+ ```console
+ export ANSIBLE_ROLES_PATH=$HOME/.ansible/collections/ansible_collections/pbench/agent/roles:$ANSIBLE_ROLES_PATH
+ ```
+
+4. Create an inventory file (`~/.config/Inventory/myhosts.inv`) naming the hosts on which you wish to install Pbench Agent and including information about where the config file. Example [inventory file](assets/myhosts.inv).
```

Comment:
Still says "including information about where the config file" ... this should be just "the location of the config file"...

Reply:
done